src/style.css

```diff
@@ -0,0 +1,145 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+/* font-family: 'Roboto', sans-serif; */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.img {
+  padding: 29px 0 41px;
+}
+
+.img__product {
+  width: 160px;
+  height: 134px;
+}
+
+.content {
+  margin: 0 14px 14px;
+}
+
+.content__title {
+  padding: 0 0 6px;
+  margin: 0;
```

Comment:
Misha taught us, that outer spacings have to be set only in blocks, not in elements)